projects/kit/components/input-date-range/input-date-range.component.ts

```diff
@@ -250,7 +250,7 @@ export class TuiInputDateRangeComponent
     }
 
     get nativeValue(): string {
-        return this.nativeFocusableElement ? this.nativeFocusableElement.value : '';
+        return this.nativeFocusableElement?.value ?? '';
```

Comment:
Should we use `||` for the final fallback in all changed lines ? 🤔 

```suggestion
        return this.nativeFocusableElement?.value || '';
```

Reply:
I think so too, there's no need to concretize the fallback case to nullish only, it's just a fallback.

Reply:
fixed